### PR TITLE
[3858] Add a rake task to import dttp trainees without academic years

### PR DESCRIPTION
**Pull request by archferns**
### Context

Adding a rake task, which can import all of the `importable` trainees which are provided by the attached csv list (by their dttp ids). 

To run
 
 ```bash 
bundle exec rails dttp:import
```

**Note:** We've already run this as part of the dttp import manually, but adding it to the codebase for visibility.

### Important business

* ~Does this PR introduce any PII fields that need to be overwritten or deleted in db/scripts/sanitise.sql?~
* ~Does this PR change the database schema? If so, have you updated the config/analytics.yml file and considered whether you need to send 'import_entity' events?~

### CODE REVIEW

**Comment by archferns** [comment]
[[M] Import DTTP trainees without academic years](https://trello.com/c/191Ns9Bx/3858-m-import-dttp-trainees-without-academic-years)

File: lib/tasks/dttp_import.rake [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+namespace :dttp do
+  desc "imports a csv of trainees from hppit"
```
Comment (AbigailMcP) [inline lib/tasks/dttp_import.rake:4]: Just need to update this description

Comment (archferns) [reply lib/tasks/dttp_import.rake]: :spaghetti:

Comment (AbigailMcP) [reply lib/tasks/dttp_import.rake]: nom

**Review by AbigailMcP — APPROVED** [review]

**Pull request merged**